config/fcos/v1_6_exp/validate.go

```diff
@@ -51,3 +51,17 @@ func (user GrubUser) Validate(c path.ContextPath) (r report.Report) {
 	}
 	return
 }
+
+func (c Config) Validate() (r report.Report) {
+	for _, disk := range c.Storage.Disks {
+		rootDevice := "/dev/disk/by-id/coreos-boot-disk"
+		if disk.Device != rootDevice && util.IsFalse(disk.WipeTable) {
+			for p, partition := range disk.Partitions {
+				if partition.Number == 0 {
+					r.AddOnWarn(path.New("partition_number", p), common.ErrInvalidPartitionConfig)
```

Comment:
This isn't a valid context path.  It turns out that tests don't check for this at all, so I've added that in https://github.com/coreos/butane/pull/458.  Once that PR lands, please rebase this PR on top and add the `VerifyReport` call in the test function.